### ISSUE TITLE
How to solve create process failed, GLE=2 error

### ISSUE THREAD

**Issue by bheem1007** [googleprojectzero/Jackalope#39]
[-] PROGRAM ABORT : CreateProcess failed, GLE=2.
         Location : Debugger::StartProcess(), C:\Users\admin\Desktop\Jackalope\TinyInst\Windows\debugger.cpp:1670
![Screenshot 2022-09-01 153920](https://user-images.githubusercontent.com/105930508/187889583-09d8e6e2-7000-4529-99b8-5ebde4ea84e7.png)

**Comment by ifratric** [comment]
GLE=2 means ERROR_FILE_NOT_FOUND, see https://docs.microsoft.com/en-us/windows/win32/debug/system-error-codes--0-499-

does Accel.exe exist in the current directory (or PATH)?

**Comment by bheem1007** [comment]
yes sir! Accel.exe exist in the path, but don't know why  fuzzing is not happening.
![Screenshot 2022-09-01 164945](https://user-images.githubusercontent.com/105930508/187901997-49d3082c-5997-4633-9d6f-4d2e539272c5.png)

**Comment by ifratric** [comment]
From the screenshot it looks like `Accel.exe` is in `Accel` directory. So it should be `.. -- Accel\Accel.exe -m @@`.

I also see you pointed input directory to `Accel`, but note that `-in` is meant for input fuzzing samples, not for directory containing the target.

**Comment by bheem1007** [comment]
so what should I mention after -in sir

**Comment by ifratric** [comment]
A directory containing input samples. Some valid files of the type `Accel.exe` processes. (I have no idea what it actually does).

**Comment by bheem1007** [comment]
This is the command I used,
C:\Users\admin\Desktop\Jackalope\build\Release>fuzzer.exe -in Accel -out out4  -t 1000 -delivery shmem -instrument_module Accel.exe -target_module Accel.exe -target_method fuzz -nargs 1 -cgc.dll -tiny-AES128-C.dll -add_all_inputs -iterations 10000 -persist -loop -cmp_coverage -- Accel\Accel.exe -m @@

this is the result am getting while using it ,please  help me to get the fuzzing output sir.
![Screenshot 2022-09-01 172241](https://user-images.githubusercontent.com/105930508/187907886-462afc08-ea8a-4176-9ed4-6bd5b7ee16f8.png)

**Comment by ifratric** [comment]
For the command to work like this, the target needs to implement the `fuzz()` method. If the target isn't a fuzzing harness you yourself wrote, It's not going to work. I also expect that your target doesn't understand `-m` argument, since this is my fuzzing harness specific. Unfortunately I can't help you to build a fuzzing setup for a specific target. But good starting points are to understand how your target works and what Jackalope does and which options do what (you can experiment with the provided `test.exe` sample). You might want to first experiment without persistent mode (without `-target_module` and `-target_method` flags) until you get a setup that works.

**Comment by bheem1007** [comment]
sir fuzzing is  running but errors are not identifying ..
![Screenshot (38)](https://user-images.githubusercontent.com/105930508/188369410-70905b4b-eb6d-41cd-9498-b8573f9213e2.png)
